### PR TITLE
Update lit-html docs README.md command

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -22,9 +22,10 @@ Previewing the docs requires the [App Engine Standard Environment for Python 2.7
 
 2. Install npm dependencies for the docs site.
 
-     cd lit-html/docs
-     npm install
-
+```
```
Comment (arthurevans) [inline docs/README.md]: Can you indent the whole block 4 spaces so it lines up under the step?

Comment (arthurevans) [inline docs/README.md]:
```suggestion
    ```
```

File: docs/README.md [hunk 2]
```diff
@@ -22,9 +22,10 @@ Previewing the docs requires the [App Engine Standard Environment for Python 2.7
...
+cd lit-html/docs
```
Comment (arthurevans) [inline docs/README.md]:
```suggestion
    cd lit-html/docs
```

File: docs/README.md [hunk 3]
```diff
@@ -22,9 +22,10 @@ Previewing the docs requires the [App Engine Standard Environment for Python 2.7
...
+npm install
```
Comment (arthurevans) [inline docs/README.md]:
```suggestion
    npm install
```

File: docs/README.md [hunk 4]
```diff
@@ -22,9 +22,10 @@ Previewing the docs requires the [App Engine Standard Environment for Python 2.7
...
+```
```
Comment (arthurevans) [inline docs/README.md]:
```suggestion
    ```
```

File: docs/README.md (hunk 1 above)
Comment (aomarks) [reply docs/README.md]: Done.

**Comment by aomarks** [comment]
> Thanks for the fix. Left suggestions to tweak left alignment.

Done.

**Comment by arthurevans** [comment]
👍

**Pull request merged**